counterpartylib/lib/messages/issuance.py

```diff
@@ -273,10 +340,17 @@ def parse (db, tx, message):
                 'asset_id': str(asset_id),
                 'asset_name': str(asset),
                 'block_index': tx['block_index'],
+                'asset_longname': subasset_longname,
             }
-            sql='insert into assets values(:asset_id, :asset_name, :block_index)'
+            sql='insert into assets values(:asset_id, :asset_name, :block_index, :asset_longname)'
             issuance_parse_cursor.execute(sql, bindings)
 
+    if reissuance:
```

Comment:
ah, this is juicy :D ... two issues with this block:
1. it seems it would crash counterparty-server in the event where an `invalid` error condition was set before `validate()` was called (as `reissuance` wouldn't be defined)
2. I don't even think it's necessary, if `validate()` no longer returns `reissued_asset_longname`, you can just remove it totally and use `'asset_longname': subasset_longname,` in `bindings` below